### PR TITLE
Port FAQ page

### PR DESCRIPTION
**Pull request by torztomasz**

### CODE REVIEW

File: packages/frontend2/src/app/(legacy)/faq/get-faq-items.tsx [hunk 1]
```diff
@@ -0,0 +1,433 @@
+import React, { type ReactNode } from 'react'
+import { OutLink } from '~/app/_components/out-link'
+
+export interface FaqItem {
+  question: string
+  answer: React.ReactNode
+}
+
+export function getFaqItems(): FaqItem[] {
```
Comment (imxeno) [inline packages/frontend2/src/app/(legacy)/faq/get-faq-items.tsx:9]: Let's make that a const with markdown answers, so:
- glossary links can pick it up
- we don't have hardcoded ReactNodes

File: packages/frontend2/src/app/utils/markdown/outLinksPlugin.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import type MarkdownIt from 'markdown-it'
+
+export function outLinksPlugin(md: MarkdownIt) {
+  const defaultRender =
+    md.renderer.rules.link_open ??
```
Comment (imxeno) [inline packages/frontend2/src/app/utils/markdown/outLinksPlugin.ts:5]: is there any reason this uses snake_case?

File: packages/frontend2/src/app/layout.tsx [hunk 3]
```diff
@@ -7,7 +7,7 @@ import { env } from '~/env'
 import { TRPCReactProvider } from '~/trpc/react'
 import { restoreCollapsibleNavStateScript } from './_components/nav/consts'
 
-import './globals.css'
+import './styles/globals.css'
```
Comment (imxeno) [inline packages/frontend2/src/app/layout.tsx:10]: Let's either make it _styles or move it outside app directory.

File: packages/frontend2/src/app/(legacy)/faq/page.tsx [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+export default async function Page() {
+  return (
+    <>
+      <Header />
+      <ContentWrapper>
+        <div className="flex">
+          <div className="mt-12 hidden w-72 shrink-0 flex-col gap-4 lg:flex">
+            {faqItems.map((item) => (
+              <a
+                key={questionToId(item.question)}
```
Comment (imxeno) [inline packages/frontend2/src/app/(legacy)/faq/page.tsx:26]: It's more efficient to map the array once to generate the ids than to use `questionToId` 5 times inside one file 😛

File: packages/frontend/src/styles/faq.css [hunk 5]
```diff
@@ -0,0 +1,7 @@
+.faq strong {
```
Comment (imxeno) [inline packages/frontend/src/styles/faq.css]: Can't we provide this kind of overrides directly to the markdown component?

File: packages/frontend2/src/app/layout.tsx (hunk 3 above)
Comment (torztomasz) [reply packages/frontend2/src/app/layout.tsx:10]: got there by mistake

**Review by imxeno — APPROVED** [review]

**Pull request merged**